polynote-spark/src/main/scala/org/apache/spark/sql/thief/DAGSchedulerThief.scala

```diff
@@ -0,0 +1,8 @@
+package org.apache.spark.sql.thief
+
+import org.apache.spark.scheduler.DAGScheduler
+import org.apache.spark.sql.SparkSession
+
+object DAGSchedulerThief {
```

Comment:
It turns out I have to "steal" package-private things from Spark, so this is just for that (along with the one below).

Would be happy to not have to do this, as it's a really dumb anti-pattern.

Reply:
Ugh, that's annoying...